content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -17,55 +17,70 @@ In the following steps, we assume that the Istio components are installed and up
 1. [Download the new Istio release](/docs/setup/kubernetes/download-release/)
```

Comment:
By removing the second step, you left this one orphaned. Just remove the number for it to render as normal text.
```suggestion
[Download the new Istio release](/docs/setup/kubernetes/download-release/)
```